appveyor.yml

```diff
@@ -17,12 +17,13 @@ install:
 
 build_script:
   - cargo doc --features "dox"
-  - cargo test
+  - cargo test --feature subclassing
+  - cargo test --feature subclassing,v2_58
```

Comment:
missed `S` and too new version

Reply:
Maybe first test without subclassing?

Reply:
Sorry, was wrong, test with subclassing and default version is needed really

Reply:
The version is fine, I was only missing the `s`. See the log

```
mingw-w64-i686-glib2-2.58.1-1
```